humility-core/src/core/gdb.rs

```diff
@@ -412,16 +414,24 @@ impl Core for GDBCore {
         ))
     }
 
-    fn write_word_32(&mut self, _addr: u32, _data: u32) -> Result<()> {
-        Err(anyhow!(
-            "{} GDB target does not support modifying state", self.server
-        ))
+    fn write_word_32(&mut self, addr: u32, data: u32) -> Result<()> {
+        let bytes: [u8; 4] = unsafe { transmute(data.to_le()) };
```

Comment:
Do you need the unsafe transmute.  Can you just do `data.to_le_bytes()`? https://play.rust-lang.org/?version=beta&mode=debug&edition=2015&gist=7e64bcd8d99a1628ac9b5f9666d8e3a9

Reply:
Thanks, data.to_le_bytes() works.